package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-react-hocs",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-react-hocs/issues",
-  "version": "5.12.0",
+  "version": "6.0.0",
```

Comment:
Why major version bump? Because apps need to provide the new lodash peer dep -> an update without additional action would break the app/package